src/feature_flags.ts

```diff
@@ -23,10 +26,9 @@ export const defaultFlags: Record<string, boolean> = {
 
   // Do not allow ___netlify-entry-point as function or file name
   zisi_disallow_new_entry_name: false,
-}
+} as const
 
-export type FeatureFlag = keyof typeof defaultFlags
-export type FeatureFlags = Record<FeatureFlag, boolean>
+export type FeatureFlags = Partial<Record<keyof typeof defaultFlags, boolean>>
```

Comment:
The type changes make it so that we have autocompletion and validation for all feature flags